### PR TITLE
Fix roundoff problem.

### LINKED ISSUE #6461: Improve IsotropicPlasticity
### Description of the enhancement or error report
The way the internal residual is computed is susceptible to round off error.  A simple change improves the situation.  I will do this.

### Rationale for the enhancement or information for reproducing the error
Improved convergence of the Newton solve in the material.

### Identified impact
(i.e. Internal object changes, limited interface changes, public API change, or a list of specific applications impacted)
No API change.

### PR DESCRIPTION
**Pull request by jasondhales**
### Completed each of the following items before creating a PR
- Include any relevant design information for your change
  Reorder to improve roundoff behavior.
- Follow [Coding Standards](http://mooseframework.org/wiki/CodeStandards/)
- Submit one or more [Test Cases](http://mooseframework.org/wiki/MooseTraining/testing/) or modify existing test cases
- Reference or close a specific issue (refs # or closes #)
  Closes #6461

### CODE REVIEW

File: modules/solid_mechanics/src/materials/ReturnMappingModel.C [hunk 1]
```diff
@@ -140,7 +140,12 @@ ReturnMappingModel::computeStress( const Elem & /*current_elem*/, unsigned qp,
     {
       Moose::err << iter_output.str();
     }
-    mooseError("Max sub-newton iteration hit during nonlinear constitutive model solve!");
+    std::stringstream err;
+    err << "Max sub-newton iteration hit during nonlinear constitutive model solve!";
+    err << " (";
+    err << _name;
+    err << ")";
+    mooseError( err.str() );
```
Comment (dschwen) [inline modules/solid_mechanics/src/materials/ReturnMappingModel.C]: FYI, you can just use 
```
mooseError("Max sub-newton iteration hit during nonlinear constitutive model solve! (" << _name << ")");
```
here.

Comment (jasondhales) [inline modules/solid_mechanics/src/materials/ReturnMappingModel.C]: Yes, of course.

**Comment by jasondhales** [comment]
Anything else needed to get this merged?

**Comment by permcody** [comment]
No - We'll get it in. Just been swamped today and haven't been doing code reviews. Thanks!

**Comment by permcody** [comment]
:+1:

**Pull request merged**